doc/motion_planning_pipeline/src/motion_planning_pipeline_tutorial.cpp

```diff
@@ -168,29 +194,29 @@ int main(int argc, char** argv)
   /* Check that the planning was successful */
   if (res.error_code_.val != res.error_code_.SUCCESS)
   {
-    ROS_ERROR("Could not compute plan successfully");
+    RCLCPP_ERROR(LOGGER, "Could not compute plan successfully");
     return 0;
   }
 
   // Visualize the result
   // ^^^^^^^^^^^^^^^^^^^^
-  ros::Publisher display_publisher =
-      node_handle.advertise<moveit_msgs::DisplayTrajectory>("/display_planned_path", 1, true);
-  moveit_msgs::DisplayTrajectory display_trajectory;
+  auto display_publisher = node->create_publisher<moveit_msgs::msg::DisplayTrajectory>("/display_planned_path", 1);
+  moveit_msgs::msg::DisplayTrajectory display_trajectory;
 
   /* Visualize the trajectory */
-  ROS_INFO("Visualizing the trajectory");
-  moveit_msgs::MotionPlanResponse response;
+  RCLCPP_INFO(LOGGER, "Visualizing the trajectory");
+  moveit_msgs::msg::MotionPlanResponse response;
   res.getMessage(response);
 
   display_trajectory.trajectory_start = response.trajectory_start;
   display_trajectory.trajectory.push_back(response.trajectory);
-  display_publisher.publish(display_trajectory);
-  visual_tools.publishTrajectoryLine(display_trajectory.trajectory.back(), joint_model_group);
+  display_publisher->publish(display_trajectory);
```

Comment:
Should we remove this publisher? My guess is that the `planning_pipeline` and this are both displaying the result which causes it to be displayed twice.
![double_traj_playback](https://user-images.githubusercontent.com/25058794/117174065-97467000-ad8a-11eb-9612-2b7fb3375a22.gif)